lib/matplotlib/figure.py

```diff
@@ -1025,48 +1025,50 @@ def subplots(self, nrows=1, ncols=1, sharex=False, sharey=False,
 
         Parameters
         ----------
-        nrows : int, default: 1
-            Number of rows of the subplot grid.
-
-        ncols : int, default: 1
-            Number of columns of the subplot grid.
+        nrows, ncols : int, default: 1
+            Number of rows/cols of the subplot grid.
 
         sharex : {"none", "all", "row", "col"} or bool, default: False
-            If *False*, or "none", each subplot has its own X axis.
-
-            If *True*, or "all", all subplots will share an X axis, and the x
-            tick labels on all but the last row of plots will be invisible.
-
-            If "col", each subplot column will share an X axis, and the x
-            tick labels on all but the last row of plots will be invisible.
-
-            If "row", each subplot row will share an X axis.
+            - If True, the x-axis will be shared amongst all subplots.
+            - If False, no axis will be shared amongst subplots.
+            - If a string must be one of "row", "col", "all", or "none".
+                - "all" has the same effect as True.
+                - "none" has the same effect as False.
+                - If "row", each subplot row will share a x-axis.
+                - If "col", each subplot column will share a x-axis.
```

Comment:
(Disclaimer: I am not a native English speaker)
In these 2 lines, shouldn't it be 'an x-axis' instead of 'a x-axis'?